sqlstore/src/main/resources/db/migration/V0020__qcpassed_type.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE Library ADD COLUMN qcPassedNew BOOLEAN NULL;
+UPDATE Library Set qcPassedNew = TRUE WHERE qcPassed = 'true' OR qcPassed = '1';
+UPDATE Library Set qcPassedNew = FALSE WHERE qcPassed = 'false' OR qcPassed = '0';
+ALTER TABLE Library DROP COLUMN qcPassed;
+ALTER TABLE Library CHANGE qcPassedNew qcPassed BOOLEAN NULL;
+
+ALTER TABLE Pool ADD COLUMN qcPassedNew BOOLEAN NULL;
+UPDATE Pool Set qcPassedNew = TRUE WHERE qcPassed = 'true' OR qcPassed = '1';
+UPDATE Pool Set qcPassedNew = FALSE WHERE qcPassed = 'false' OR qcPassed = '0';
+ALTER TABLE Pool DROP COLUMN qcPassed;
+ALTER TABLE Pool CHANGE qcPassedNew qcPassed BOOLEAN NULL;
+
+ALTER TABLE Sample ADD COLUMN qcPassedNew BOOLEAN NULL;
```

Comment:
`BOOLEAN` is the right type to use. `TINYINT` is 1 byte, so it can vary be more than 0 or 1. `BIT(1)` does work, but since mySQL treats it as a special type, it likes to dump special characters to output. that breaks compatibility with H2 and is generally a pain on the console.